includes/gateways/paypal-standard.php

```diff
@@ -693,8 +695,9 @@ function give_get_paypal_button_type() {
  * @return string
  */
 function give_build_paypal_item_title( $payment_data ) {
```

Comment:
Add a filter for the return output named `give_payment_gateway_item_title` and also pass the `$form_id` and `$payment_data` as params.

Reply:
@DevinWalker Added Filter `give_payment_gateway_item_title` with proper comments